superbench/common/utils/process.py

```diff
@@ -4,19 +4,43 @@
 """Process Utility."""
```

Comment:
How to enable it by user?

Reply:
user can add a parameter to enable it for any benchmark in config, pls check #445

Reply:
Thanks